app/Validators/NewEventValidator.ts

```diff
@@ -0,0 +1,31 @@
+import { schema, CustomMessages, rules } from '@ioc:Adonis/Core/Validator'
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+
+export default class NewEventValidator {
+  constructor(protected ctx: HttpContextContract) {}
+
+  public schema = schema.create({
+    title: schema.string([rules.unique({ table: 'events', column: 'title' })]),
+    description: schema.string.optional(),
+    box_office: schema.string.optional(),
+    pre_payment: schema.string.optional(),
+    alternative_address: schema.string.optional(),
+    cover: schema.string.optional([rules.url()]),
+    creator_email: schema.string.optional([
+      rules.email(),
+      rules.exists({ table: 'users', column: 'email' }),
+    ]),
+    venue_id: schema.string.optional([
+      rules.uuid(),
+      rules.exists({ table: 'venues', column: 'id' }),
+    ]),
+    links: schema.string.optional(),
+  })
+
+  public messages: CustomMessages = {
+    '*': (field, rule) => {
+      return `${rule} validation error on ${field}`
+    },
+    'title.required': 'title is required to create an event',
+  }
+}
```

Comment:
Ist das erste nicht standard? Oder muss man das überschreiben?

Ist die webseite auf deutsch? Dann sollten wir auch die Fehlermeldungen auf deutsch anzeigen.

und:
sollten wir mehr Felder requiren?

Reply:
Sollten wir vielleicht mehr Felder requiren?

Reply:
1. ich hatte den fall das beim fehlenden title da stand "undefined is required" oder so. aber vielleicht was das auch nur weil ich im string die rule required dring hatte "rules.required()"

Reply:
2. ich bin mir da jetzt nicht mehr sicher für was wir und entschieden haben.

Reply:
3. Wir hatten glaube ich gesagt das title, headliner und der tag/zeit Pflicht ist. die fehlenden Felder müsste man noch in ein eigenen pr machen oder? (headliner: eine artist; support: array<artist>; datetime: iso...

Reply:
3. Alles klar. Ja auf jeden Fall in nem extra PR. 
Und irgwendwo dokumentieren :pray: Habe gerade nochmal ins GDrive geschaut und dazu nichts gefundne